### DISCUSSION TITLE
How to Use LaTeX Partials Since Pandoc 3.5

### DISCUSSION THREAD

**Discussion by TomBener** [quarto-dev/quarto-cli#10988] [Q&A] [answered]
### Description

[Pandoc 3.5](https://github.com/jgm/pandoc/releases/tag/3.5) split out common parts of latex template into partials: `common.latex`, `fonts.latex`, `font-settings.latex`, `passoptions.latex`, `hypersetup.latex`, and `after-header-includes.latex`. And the [default LaTeX template](https://github.com/jgm/pandoc/blob/main/data/templates/default.latex) also changed significantly in this version. Since Quarto itself has [partials](https://quarto.org/docs/journals/templates.html#latex-partials) as well, I’m confused how these changes will affect using LaTeX partials in Quarto. Thank you!

**Comment by cderv** [comment, marked as answer]
Currently Quarto 1.5 is using Pandoc 3.2, and next Quarto 1.6 will be using Pandoc 3.4. So regarding using Pandoc 3.5 new templates, this is not a concerns with current versions of Quarto. 

We'll probably handle the Pandoc 3.5 update for Quarto 1.7. When the time comes, we'll handle the merging of the template. Currently I can't say if Pandoc change will have such impact that it will lead to breaking change, but I am hopeful that there won't be any change in our current partial system. They would probably just used new partials. We use our own copy of template for different format, and we'll continue to do the same while following relevant Pandoc's update in the template. 

Anyhow, we can't say much for now as we are not currently working on this. Quarto usually has less breaking change than Pandoc and obvioulsy all effort will be made to keep features working. I'll create an issue to track this item for next version. 

Hope this helps

**Reply by cderv** [reply]
Tracked now at 
- https://github.com/quarto-dev/quarto-cli/issues/10990
